{
  "channeldata": {
    "activate.d": false,
    "binary_prefix": false,
    "deactivate.d": false,
    "description": null,
    "dev_url": null,
    "doc_source_url": null,
    "doc_url": null,
    "home": "https://github.com/pytest-dev/pytest-faulthandler",
    "icon_hash": null,
    "icon_url": null,
    "identifiers": null,
    "keywords": null,
    "license": "MIT",
    "post_link": false,
    "pre_link": false,
    "pre_unlink": false,
    "recipe_origin": null,
    "run_exports": {},
    "source_git_url": null,
    "source_url": "https://pypi.io/packages/source/p/pytest-faulthandler/pytest-faulthandler-1.5.0.tar.gz",
    "subdirs": [
      "win-64"
    ],
    "summary": "py.test plugin that activates the fault handler module for tests",
    "tags": null,
    "text_prefix": false,
    "timestamp": 0,
    "version": "1.5.0"
  },
  "channeldata_version": 1,
  "feedstock": null,
  "labels": [
    "main",
    "cf202003"
  ],
  "package": "pytest-faulthandler-1.5.0-py35_0.tar.bz2",
  "repodata": {
    "build": "py35_0",
    "build_number": 0,
    "depends": [
      "pytest",
      "python 3.5*"
    ],
    "license": "MIT",
    "md5": "f7e87293ea3e3a860ecbe2bcb45bfde4",
    "name": "pytest-faulthandler",
    "sha256": "cc5c38872a0a6d3b13223a64aa68799caef781c40774a6f6b983840184b0c826",
    "size": 8725,
    "subdir": "win-64",
    "version": "1.5.0"
  },
  "repodata_version": 1,
  "subdir": "win-64",
  "url": "https://conda.anaconda.org/conda-forge/win-64/pytest-faulthandler-1.5.0-py35_0.tar.bz2"
}